kasa/tests/test_feature_brightness

```diff
@@ -0,0 +1,12 @@
+from kasa.smart import SmartDevice
+
+from .conftest import (
+    brightness_smart,
+)
+
+
+@brightness_smart
```

Comment:
Maybe already add this inside `kasa/tests/smart`, or even `kasa/tests/smart/modules`?

Reply:
I think we probably want to end up with features that are common interfaces across the smart and iot devices rather than having separate tests for each no?

Reply:
True, but we may still want to have tests for the family-specific payloads. But if this is reserved for the common brightness API, maybe this should rather be `@brightness`? This would be where we would test the `set_brightness` and `brightness`, leaving the module-specifics in their own test files?

Reply:
Yes I think so.  This change isn't actually implementing the testing, it's checking the component filters work until we add real tests.

Have renamed to `brightness`